support/itemUtils.ts

```diff
@@ -149,41 +142,39 @@ export function getItemTitle(item: PortalItem): string {
 
 export function goToMarker(
   marker: string,
-  view: MapView | SceneView
-): IPromise<any> {
+  view: esri.MapView | esri.SceneView
+): Promise<any> {
   if (!marker || !view) {
     return promiseUtils.resolve();
   }
 
   return parseMarker(marker).then(graphic => {
-    view.graphics.add(graphic);
-    const view2 = view as any; // todo: Typings will be fixed in next release.
-    return view2.goTo(graphic);
+    view.graphics.add(graphic as __esri.Graphic);
+    view.watch("ready", () => {
```

Comment:
Can we use view.when() async await here?

Reply:
done